components/brave_wallet_ui/page/screens/onboarding/connect-hardware/onboarding-connect-hardware-wallet.tsx

```diff
@@ -38,10 +50,24 @@ export const OnboardingConnectHardwareWallet = () => {
   )
 
   return (
-    <WalletPageLayout>
-      <OnboardingWrapper>
-        <AddHardwareAccountModal onSelectAccountType={onSelectAccountType} />
-      </OnboardingWrapper>
-    </WalletPageLayout>
+    <OnboardingContentLayout
+      title={getLocale('braveWalletConnectHardwareWalletSelectBlockchain')}
+      padding='56px 0 0'
+      showBackButton={false}
+    >
+      {accountOptions.map((option, index) => {
+        return (
+          <React.Fragment key={option.name}>
+            <AccountType
+              title={option.name}
+              description={option.description}
+              icons={option.chainIcons ? option.chainIcons : []}
```

Comment:
nit: can be simplified
```suggestion
              icons={option.chainIcons || []}
```